RInAppMessaging/Classes/Models/Responses/PingResponse.swift

```diff
@@ -27,6 +27,11 @@ internal struct Campaign: Decodable, Hashable {
         let endTimeMilliseconds = data.messagePayload.messageSettings.displaySettings.endTimeMilliseconds
         return endTimeMilliseconds < Date().millisecondsSince1970
     }
+    var contexts: [String] {
+        data.messagePayload.title.components(separatedBy: "]").dropLast().map { substring in
```

Comment:
I tried to use regex but with creation of NSRegularExpression object the current solution is faster

Reply:
yeah this hurt my head a bit 😄